plugin.php

```diff
@@ -5,33 +5,60 @@
-        update_site_option('zencache_apc_warning_bypass', time());
+    if (!empty($_REQUEST['zencache_min_php54_warning_bypass']) && is_admin()) {
+        update_site_option('zencache_min_php54_warning_bypass', time());
     }
-    if (extension_loaded('apc') && filter_var(ini_get('apc.enabled'), FILTER_VALIDATE_BOOLEAN) && filter_var(ini_get('apc.cache_by_default'), FILTER_VALIDATE_BOOLEAN) && stripos((string) ini_get('apc.filters'), 'zencache') === false && !get_site_option('zencache_apc_warning_bypass')) {
-        if (empty($_REQUEST['zencache_apc_warning_bypass']) && is_admin()) {
-            ${__FILE__}['apc_warning'] = '<h3 style="margin:.5em 0 .25em 0;">'.__('<strong>APC EXTENSION WARNING</strong></h3>', 'zencache');
-            ${__FILE__}['apc_warning'] .= '<p style="margin-top:0;">'.sprintf(__('<strong>ZenCache says...</strong> It appears that you\'re currently running PHP v%1$s with APC enabled. APC is <a href="http://zencache.com/r/apc-compatibility/" target="_blank">known to contain bugs</a>.', 'zencache'), esc_html(PHP_VERSION)).'</p>';
+    if (!empty($_REQUEST['zencache_apc_deprecated_warning_bypass']) && is_admin()) {
+        update_site_option('zencache_apc_deprecated_warning_bypass', time());
+    }
+    $_apc_enabled = (extension_loaded('apc') && filter_var(ini_get('apc.enabled'), FILTER_VALIDATE_BOOLEAN) && filter_var(ini_get('apc.cache_by_default'), FILTER_VALIDATE_BOOLEAN) && stripos((string)ini_get('apc.filters'), 'zencache') === false) ? true : false;
+    if (!version_compare(PHP_VERSION, '5.4', '>=')) {
+        if (empty($_REQUEST['zencache_min_php54_warning_bypass']) && is_admin()) {
+            ${__FILE__}['php54_warning'] = '<h3 style="margin:.5em 0 .25em 0;">'.__('<strong>NOTICE: ZenCache Minimum PHP Version</strong></h3>', 'zencache');
+            ${__FILE__}['php54_warning'] .= '<p style="margin-top:0;">'.sprintf(__('As of December 1st, 2015 ZenCache will require PHP 5.4 or higher.</strong> The minimum PHP version required to run ZenCache is being increased from PHP 5.3.2+ to PHP 5.4+. It appears that your server is currently running PHP v%1$s. You will need to upgrade to PHP 5.4 or higher to run the next version of ZenCache.', 'zencache'), esc_html(PHP_VERSION)).'</p>';
+            ${__FILE__}['php54_warning'] .= '<p style="margin-top:0;">'.__('To learn more about this upcoming change, please see the announcement: <a href="http://zencache.com/r/new-minimum-php-version-php-5-4/" target="_blank">New Minimum PHP Version: PHP 5.4</a>', 'zencache').'</p>';
+            if ($_apc_enabled) {
+                ${__FILE__}['php54_warning'] .= '<p style="margin-top:0;">'.__('Additionally, it appears that your server is also running the outdated PHP APC extension. <strong>The PHP APC extension will also no longer be supported as of December 1st, 2015.</strong> Please see: <a href="">PHP APC Extension No Longer Supported</a>', 'zencache').'</p>';
```

Comment:
Aha! I see what you mean now; i.e., `<a href="">`. Yep, I missed that before. Sorry.